lib/settings_ui.dart

```diff
@@ -4,4 +4,3 @@ export 'package:settings_ui/src/settings_section.dart';
 export 'package:settings_ui/src/settings_tile.dart';
 export 'package:settings_ui/src/settings_list.dart';
 export 'package:settings_ui/src/custom_section.dart';
-
```

Comment:
I think it'll be good to follow the `dartfmt` convention by adding the EOF empty line.

Reply:
Indeed, but there were 2 lines ;) So one is left.

Reply:
Sure. That's fine then